README.md

```diff
@@ -2141,6 +2142,92 @@ def reply_raw() -> manual[RawReply]:
 
 [Not yet implemented.](https://github.com/demergent-labs/azle/issues/496)
 
+### Timer APIs
+
+-   [clear timer](#clear-timer)
+-   [set timer](#set-timer)
+-   [set timer interval](#set-timer-interval)
+
+#### clear timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
+```python
+from kybra import ic, TimerId, update
+
+@update
+def clear_timer(timer_id: TimerId):
+    ic.clear_timer(timer_id)
+```
+
+#### set timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
+```python
+from kybra import Duration, ic, TimerId, update
+
+
+@update
+def set_timers(delay: Duration) -> list[TimerId]:
+    global_function_timer_id = ic.set_timer(delay, global_function)
+
+    lambda_timer_id = ic.set_timer(
+        delay,
+        lambda: ic.print("inline lambda called"),
+    )
+
+    captured_value = "🚩"
```

Comment:
Let's unify the documentation, this example is different from the other example. I like some of the things this example is showing, but the other example should probably show them too.

Also I'm not sure why you have an inner function in here, you should be able to just have a callback defined like in JS outside of this function.